<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>892</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>97</red>
        <green>53</green>
        <blue>131</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>97</red>
        <green>53</green>
        <blue>131</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>97</red>
        <green>53</green>
        <blue>131</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>97</red>
        <green>53</green>
        <blue>131</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>94</red>
         <green>92</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>94</red>
         <green>92</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>94</red>
         <green>92</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>94</red>
         <green>92</green>
         <blue>100</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Circuit Altitude (ft)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="circuit_altitude">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Chracteristic Length (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Char_Len">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Flow Velocity (m/s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Flow_Velocity">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pressure">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Density">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Temperature">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Pressure (Pa)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Density (kg/cubic m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature (K)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Viscosity">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Kine. Viscosity (m2/s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ReynoldsNo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reynolds Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RANS PARAMETER TUNER FOR OPENFOAM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReynCalc">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>670</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecordData1">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>800</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Record Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewFile">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>New File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFile">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="count">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>800</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="skinFriction">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wall Skin Friction Coeff.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shearStress">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wall Shear Stress (Pa)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yPlus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>y+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cellHeight">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cell Height (m)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>730</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="noLayers">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="expRatio">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No. of Layers</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Expansion Ratio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="layerHeight">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer Height (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BLayerHeight">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B. Layer Thickness(m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="blockSize">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Background Cell Size (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="refineLevel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mesh Refinement Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Flow Conditions (International Standard Atmosphere Model)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>901</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Boundary Layer and Mesh Refinement (Schlichting Turbulent Boundary Layer Model)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OpenSymbol</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turbulence Parameters (2-Equation Models)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="turbIntensity">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turbulence Intensity (%)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Turbulent K.E. (m2/s2)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="turbKE">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>560</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>560</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Omega (per s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="turbOmega">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>560</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>- Bold Parameters are required to proceed.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>- Bold Italic Parameters are optional but atleast one parameter from each section must be entered</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>840</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>- Data can only be recorded after creating a new file in the current session</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>- File can only be opened after it's created in the current session</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>750</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IMPORTANT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>- Only applicable for turbulent RANS simulations</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="turbEpsilon">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>590</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>590</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Epsilon (m2/s3)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
